### PR TITLE
Use sparse matrix to implement FusedEmbeddingSeqPoolGradKernel 

### PR DESCRIPTION
**Pull request by zhaify**
This PR is the  implementation of `FusedEmbeddingSeqPoolGradKernel`  with *MKL* sparse matrix to save the cost of memory loading/transferring, since PR [19064](https://github.com/PaddlePaddle/Paddle/pull/19064) has implemented the `FusedEmbeddingSeqPoolKernel` OP with *MKL* sparse matrix before.

### CODE REVIEW

**Comment by bingyanghuang** [comment]
@yihuaxu Please help review this PR.

File: paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h [hunk 1]
```diff
@@ -31,6 +33,43 @@ using LoDTensor = framework::LoDTensor;
 using SelectedRows = framework::SelectedRows;
 using DDim = framework::DDim;
 
+#if defined(PADDLE_WITH_MKLML) && !defined(_WIN32) && !defined(__APPLE__) && \
```
Comment (tensor-tang) [inline paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h]: Same as here https://github.com/PaddlePaddle/Paddle/pull/19064#discussion_r314566111

**Comment by luotao1** [comment]
please update this PR since #19064 is merged @bingyanghuang @zhaify

**Comment by luotao1** [comment]
- 这个fuse op的embedding那步不支持传入padding_idx？
- 对应我们框架embedding(也就是lookup_table op)里的padding_idx。对于输入的word_id如果等于padding_idx，则输出全零向量

File: paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h (hunk 1 above)
Comment (zhaify) [reply paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h]: done

**Comment by zhaify** [comment]
> * 这个fuse op的embedding那步不支持传入padding_idx？
> * 对应我们框架embedding(也就是lookup_table op)里的padding_idx。对于输入的word_id如果等于padding_idx，则输出全零向量

目前不支持 `padding_idx`我们后续会改下`fused_emb_seq`这个`op` 的前向和反向，会提新的PR

**Comment by yihuaxu** [comment]
LGTM

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**